### ISSUE TITLE
About BatchNormalization

### ISSUE THREAD

**Issue by RainPot** [CoinCheung/DeepLabv3plus#3]
hi
by running your project,I can reach high mIoU,but when the Batchnormlization is setting all SYNC in ResNet,the mIoU can just reach 70%,why is that?

**Comment by RainPot** [comment]
thanks for great code btw

**Comment by sonack** [comment]
@Cynicsss  How did you control the BN sync or not sync? I found the bn in resnet is original all sync already? What do you mean by `,but when the Batchnormlization is setting all SYNC in ResNet,the mIoU can just reach 70%`

**Comment by RainPot** [comment]
> @Cynicsss How did you control the BN sync or not sync? I found the bn in resnet is original all sync already? What do you mean by `,but when the Batchnormlization is setting all SYNC in ResNet,the mIoU can just reach 70%`

https://github.com/CoinCheung/DeepLab-v3-plus-cityscapes/blob/e31cc7567f87c44e7ef449eec738e8bb18c3605a/models/resnet.py#L51

**Comment by sonack** [comment]
> > @Cynicsss How did you control the BN sync or not sync? I found the bn in resnet is original all sync already? What do you mean by `,but when the Batchnormlization is setting all SYNC in ResNet,the mIoU can just reach 70%`
> 
> https://github.com/CoinCheung/DeepLab-v3-plus-cityscapes/blob/e31cc7567f87c44e7ef449eec738e8bb18c3605a/models/resnet.py#L51

Do you know why the author need to use the original `nn.BatchNorm2d`rather than `synced bn` here ?

**Comment by codermckee** [comment]
@Cynicsss Hi, how long did it take you to reach 70% mIoU?   I tried the code, but it takes one night just training about 12000 iterations.

**Comment by codermckee** [comment]
@Cynicsss I use 3 *1080ti to train on CityScapes without any change, but just reaching 70.23% mIoU. Is there something I should change in order to reach higher mIoU? BTW, it costed me about 2.5 days to train 41000 iterations..so sad.

**Comment by LetsGoFir** [comment]
> @Cynicsss I use 3 *1080ti to train on CityScapes without any change, but just reaching 70.23% mIoU. Is there something I should change in order to reach higher mIoU? BTW, it costed me about 2.5 days to train 41000 iterations..so sad.

It takes me 4 hours to train 15000 iterations with 2 RTX2080 and finally I got 78.2 mIoU.

**Comment by prachigarg23** [comment]
@LetsGoFir can you please confirm if this repository is reproducible for cityscapes? I'm also using 2RTX 2080 GPUs and facing issues getting 75%+ mIoU on CS. Also any idea if there are any differences between this implementation and the official implementation?
